src/components/VmUserMessages/style.css

```
@@ -1,7 +1,21 @@
 .panel-body {
-    border-bottom: 1px solid #d1d1d1;
     padding: 0;
     background-color: #fff;
+    overflow-y: auto;
+    display: flex;
+    flex-direction: column;
+}
+
+.drawer-pf-notifications-list {
+    border: solid 1px #d1d1d1 !important;
+    padding: 10px 0 !important;
+    margin: 0;
+    overflow-y: auto;
+}
+
+.drawer-pf-action-clear-notification-list {
+    flex: auto;
+    overflow-y: auto;
```

```suggestion
.action-panel {
    border-top: solid 1px #d1d1d1;
```

Only the extra border style is needed if the class itself is attached to the parent component it is currently attached to.